docs/content/docs/troubleshooting/traffic/ingress.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Ingress Troubleshooting"
+description: "Ingress Troubleshooting Guide"
+type: docs
+aliases: ["ingress.md"]
+---
+
+## When Ingress is not working as expected
+
+### 1. Confirm global ingress configuration is set as expected. 
+
+```console
+# Returns true if HTTPS ingress is enabled
+$ kubectl get ConfigMap osm-config -n osm-system -p jsonpath='{.data.use_https_ingress}{"\n"}'
+```
+
+If the output of this command is `false` this means that HTTP ingress is enabled and HTTPS ingress is disabled. To disable HTTP ingress and enable HTTPS ingress, use the following command: 
+
+> Note: To make this change persistent between upgrades, see osm mesh upgrade --help.
+```bash
+# Replace osm-system with osm-controller's namespace if using a non default namespace
+kubectl patch ConfigMap osm-config -n osm-system -p '{"data":{"use_https_ingress":"true"}}' --type=merge
+```
+
+Likewise, to enable HTTP ingress and disable HTTPS ingress, run: 
```

Comment:
run what?

Reply:
Forgot to delete the line under this. It should just be the cmd directly below.